code/mission/missionparse.cpp

```diff
@@ -6002,6 +6004,32 @@ void parse_sexp_containers()
 	}
 }
 
+void parse_custom_data(mission* pm)
+{
+	if (!optional_string("#Custom Data"))
+		return;
+
+	if (optional_string("$begin_data_map")) {
+
+		parse_string_map(pm->custom_data, "$end_data_map", "+Val:");
+	}
+}
+
+void apply_default_custom_data(mission* pm)
+{
+	for (const auto& def : Default_custom_data) {
+		int count = 0;
+		for (const auto& listed : pm->custom_data) {
+			if (listed.first != def.key) {
+				count++;
+			}
+		}
+		if (count == (int)pm->custom_data.size()) {
```

Comment:
For new code, we should use c++ style casts.  In this case `static_cast<int>(pm->custom_data.size()` should work fine.  I know it sounds picky, but c style casts can accidentally do unintended things, like remove const or use reinterpret cast if we're not careful. Also c++ style casts are easier to search for and therefore audit.

Reply:
Converted the ones that needed to stay as Ints because FRED. Changed one or two to size_t. Deleted the others because those functions are actually unused leftovers from copy/paste the containers dialog. Should be good to go. 👍